### PR TITLE
Format table cells with dashboard click behavior as links

### LINKED ISSUE #13548: Difficult to see which cells has "Click behavior" vs normal behavior
**Is your feature request related to a problem? Please describe.**
Since 0.37, it's possible to create custom drill-through destinations, but it's difficult to see from table cells (and possibly other visualizations) what will happen when they are clicked.
Example, which of these would be considered clickable? (hint, all of them have custom click behavior set to URL)
image

**Describe the solution you'd like**
Possibly set similar indication as normal URLs or PK/FK.

**Describe alternatives you've considered**
Using "Customize link text" to add something like `Go to link for {{column}}`, but that will make the columns much wider and will still leave normal URLs and PK/FK with a lot more visual hint.

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

### PR DESCRIPTION
**Pull request by paulrosenzweig**
Resolves #13548

This PR updates the formatting code for these cells to wrap the text in a div with `className="link link--wrappable"`. Those classes are what we had on the actual links used prior to v0.37.

### Before
image

### After
image

### CODE REVIEW

File: frontend/src/metabase/lib/formatting.js [hunk 1]
```diff
@@ -740,6 +740,18 @@ export function formatValueRaw(value: Value, options: FormattingOptions = {}) {
 
   if (value == null) {
     return null;
+  } else if (
+    options.click_behavior &&
+    clickBehaviorIsValid(options.click_behavior) &&
+    options.jsx
+  ) {
+    // Style this like a link if we're in a jsx context.
+    // It's not actually a link since we handle the click differently for dashboard and question targets.
```
Comment (paulrosenzweig) [inline frontend/src/metabase/lib/formatting.js:749]: Just FYI for any readers, I think these could be actual links but there's a bit more nuance in generating the URLs. We need to make sure that we've loaded data about the target dashboard/question and have a way to encode GUI question parameters in the URL.

**Comment by camsaul** [comment]
That snowflake failure is actually a different one from the quota issue -- I have a fix for it in #15001

**Review by kdoh — APPROVED** [review]

**Comment by flamber** [comment]
This now also adds styling to public/embed links to dashboard/question, which does not work, but makes it harder to understand that you cannot click the text.

**Comment by paulrosenzweig** [comment]
@flamber That sounds like a less severe bug. Do you think it makes sense to do a "bug trade" by merging this and opening an issue for that?

**Comment by flamber** [comment]
@paulrosenzweig Definitely merge this and then we'll open another for public/embedding.

**Pull request merged**